policyengine_us/parameters/gov/states/ma/tax/income/credits/senior_circuit_breaker/amount/max.yaml

```
@@ -4,14 +4,20 @@ values:
   # COLA-adjusted, rounded to next lowest multiple of $10.
   2021-01-01: 1_170
   2022-01-01: 1_200
+  2023-01-01: 2_590
 metadata:
   unit: currency-USD
   period: year
   label: MA Senior Circuit Breaker maximum payment
   reference:
-    - title: Mass. General Laws c.62 § 6(k)
+    # The legal code specifies infaltion unadjusted amounts
```

Specify the new legal code value since it changed for 2023
```suggestion
    # The legal code specifies inflation unadjusted amounts.
```